### PR TITLE
Opacity control for rasters

### PR DESCRIPTION
**Pull request by theduckylittle**
1. Adds styling for raster opacity.
2. Adds a demo of the raster-opacity to the basic demo.

refs: SDK-906

### CODE REVIEW

File: examples/basic/app.js [hunk 1]
```diff
@@ -167,6 +171,43 @@ function main() {
...
+            defaultValue={opacity}
+            ref={(me) => {
+              this.opacityInput = me;
+            }}
+          />
+
+          <button
+            className="sdk-btn"
+            onClick={() => {
+              const v = this.opacityInput.value / 100;
```
Comment (wnordmann) [inline examples/basic/app.js]: `v` is a vague variable name

File: src/components/map.js [hunk 2]
```diff
@@ -860,6 +860,12 @@ export class Map extends React.Component {
       applyStyle(olLayer, fake_style, layers[0].source);
     }
 
+    if (layers.length === 1 && layers[0].type === 'raster') {
```
Comment (wnordmann) [inline src/components/map.js]: This line is inside a forloop, if` layers.length === 1` then `layers[0]  === layer`, use layer instead

Comment (theduckylittle) [reply src/components/map.js]: This isn't inside a for loop. The for-loop ends on line 848.  With the way raster layers are entered, they only have one layer per source, so the loop isn't necessary.

**Comment by theduckylittle** [comment]
@wnordmann - Updated!

**Pull request merged**